.gitignore

```diff
@@ -2,6 +2,7 @@
 target
 tmp.discovery
 **/.idea
+**/.vscode
```

Comment:
Hi @liquidslr, We typically try to avoid making changes to files that don't specifically pertain to the PR. In this case I would make a [global .gitignore file ](https://help.github.com/en/articles/ignoring-files) on your machine so that this file is unchanged.

Reply:
Sure @scottcarol I will revert the change